presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -131,6 +131,8 @@ protected String toSqlType(Type type)
 
         String sqlType = super.toSqlType(type);
         switch (sqlType) {
+            case "real":
+                return "float";
```

Comment:
Will MySQL's `float` be reported by MySQL JDBC as `j.sql.Types.REAL` so that `toPrestoType` would return `com.facebook.presto.spi.type.RealType#REAL` ?
I.e. if i create table `mysql.xxx.t(a real)` in Presto, will it show up as having column of type `real`?

(I already forgot whether `DataTypeTest` covers this :) )

Reply:
The newly added `testBasicTypes()` should cover this, otherwise it wouldn't round-trip (unless it is broken in both directions).